src/app/(auth)/signup/page.tsx

```diff
@@ -1,25 +1,27 @@
 'use client';
 
 import { useState } from 'react';
-import { useRouter } from 'next/navigation';
+// import styled from 'styled-components';
+// import { useRouter } from 'next/navigation';
 import TextInput from '@/components/TextInput/index';
-import { H1, H4, AColored } from '@/styles/text';
+import { H1, H2, H4, AColored, P } from '@/styles/text';
 import supabase from '@/api/supabase/createClient';
 import COLORS from '@/styles/colors';
 import {
   OuterDiv,
   SpacerDiv,
+  HorizontalDiv,
   FormDiv,
   H4Centered,
-  QuestionsDiv,
 } from '@/app/(auth)/styles';
 import BigButton from '@/components/BigButton';
+import Button from '@/components/Button';
 
 export default function SignUp() {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  // const [errorMessage, setErrorMessage] = useState('');
-  const { push } = useRouter();
+  const [emailSentCount, setEmailSentCount] = useState(0);
+  // const { push } = useRouter();
   const handleSignUp = async () => {
     const { error } = await supabase.auth.signUp({
       email,
```

Comment:
AGH my bad this was updated in the Figma and I forgot to update in the Linear issue, but we also want to make this page (on, say `/email-verified`)

<img width="661" alt="Screenshot 2023-11-18 at 3 08 18 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/b464205e-9c50-4323-950d-6412185389d1">

And once you do, you want to come back here and supply the route to this new page as the redirect URL:

```

const { data, error } = await supabase.auth.signUp(
  {
    email: 'example@email.com',
    password: 'example-password',
    options: {
      emailRedirectTo: /** add link here */
    }
  }
)
```